src/test/java/org/jitsi/videobridge/rest/root/colibri/debug/DebugTest.java

```diff
@@ -75,7 +89,16 @@ public void testEnableNonexistentDebugFeature()
     @Test
     public void testDisableDebugFeature()
     {
-        Response resp = target(BASE_URL + "/disable/" + DebugFeatures.PAYLOAD_VERIFICATION.getValue())
+        Response resp = target(BASE_URL + "/foo/bar/disable/" + DebugFeatures.PAYLOAD_VERIFICATION.getValue())
```

Comment:
Can you add some negative test cases as well? E.g.: one where the state isn't valid and one where the feature name isn't valid.

Reply:
done